tests/window/test_partition_only.py

```diff
@@ -345,6 +345,48 @@ def test_window_value_over_sum(make_df):
     assert_df_equals(result.to_pandas(), pd.DataFrame(expected), sort_key=list(expected.keys()))
 
 
+@pytest.mark.skipif(get_tests_daft_runner_name() != "native", reason="Window tests only run on native runner")
```

Comment:
Let's put this as a top level skip on the file itself. Look at the usage of the variable `pytestmark` in our other tests for an example of how to do that.